app/src/main/java/gov/va/vro/config/MasVeteranFlashConfig.java

```diff
@@ -0,0 +1,17 @@
+package gov.va.vro.config;
+
+import gov.va.vro.model.mas.MasVeteranFlashProps;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/** Configure Veteran Flash IDs. */
+@Configuration
+public class MasVeteranFlashConfig {
+
+  @Bean
+  MasVeteranFlashProps MasVeteranFlashConfigLoad(
+      @Value("${masVeteranFlashIds.agentOrange}") String[] agentOrangeFlashIds) {
```

Comment:
MasVeteranFlashProps has been defined as a singleton, hasn't it? Is @Bean default to a singleton already?